packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -280,8 +280,9 @@ class ReorderableList extends StatefulWidget {
 
   /// {@macro flutter.widgets.scroll_view.keyboardDismissBehavior}
   ///
-  /// The default is [ScrollViewKeyboardDismissBehavior.manual]
-  final ScrollViewKeyboardDismissBehavior keyboardDismissBehavior;
+  /// If [keyboardDismissBehavior] is null then it will fallback to the closest
+  /// [ScrollConfiguration] provided by the context.
```

Comment:
```suggestion
  /// If [keyboardDismissBehavior] is null then it will fallback to the inherited
  /// [ScrollBehavior.keyboardDismissBehavior].
```